Entities/Scene.cpp

```diff
@@ -2878,8 +2890,10 @@ int Scene::SetOwnerOfAllDoors(int team, int player)
 
 void Scene::ResetPathFinding()
 {
-    if (m_pPathFinder)
-        m_pPathFinder->RecalculateAllCosts();
+    for (int i = 0; i < sc_pathfinderCount; ++i) {
+        // Update all the pathfinding data
```

Comment:
Please use a range for loop here, no reason for a manual one.

Also, I don't think this comment adds anything the code doesn't already say, so it can be removed.